### DISCUSSION TITLE
When multiple targets of a project are built together, does it upload the remote cache during the build process, or wait until all builds are completed before uploading?

### DISCUSSION THREAD

**Discussion by Eric-hust** [bazelbuild/bazel#22736] [Q&A] [answered]
When multiple targets of a project are built together, does it upload the remote cache during the build process, or wait until all builds are completed before uploading?

**Comment by brentleyjones** [comment, marked as answer]
As soon as any action is done (a target can be made up of many actions), it starts uploading to the remote cache.

**Comment by Eric-hust** [comment]
@brentleyjones Is there any way to check the status of upload and download? I checked the build options and it seems that there is no option to check the status.

**Reply by brentleyjones** [reply]
`--remote_grpc_log` and https://github.com/bazelbuild/tools_remote

**Reply by Eric-hust** [reply]
Thank you for your answer！
